### PR TITLE
all655 - Code Review Changed Completed

### PR DESCRIPTION
**Pull request by all655**
Hey, I made the two changes that Ethan suggested: a spacing error with the code blocks and a grammar error.

### CODE REVIEW

File: ESD109-Shadowsedit/ShadowsOfTheKnight_all655.cpp [hunk 1]
```diff
@@ -0,0 +1,183 @@
+// # Assignment: Introduction to the CodeChat Editor
```
Comment (bjones1) [inline ESD109-Shadowsedit/ShadowsOfTheKnight_all655.cpp:1]: Place this in the correct directory: name of the game/netid/filename.cpp.

File: ESD109-Shadowsedit/ShadowsOfTheKnight_all655.cpp [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+// ### Name: Lael Lum
+//
+// ### NetID: all655
+//
+// ---
+//
+// ## The puzzle I chose was *<a title="Shadows of the Knight - Episode 1" href="https://www.codingame.com/training/medium/shadows-of-the-knight-episode-1" target="_blank" rel="noopener">Shadows of the Knight - Episode 1</a>*
+//
+// ### How I went about solving this puzzle: 
+//
```
Comment (bjones1) [inline ESD109-Shadowsedit/ShadowsOfTheKnight_all655.cpp:14]: Too much empty space -- use just 1 line between paragraphs.

File: ESD109-Shadowsedit/ShadowsOfTheKnight_all655.cpp [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+using namespace std;
+
+/**
+ * Auto-generated code below aims at helping you parse
+ * the standard input according to the problem statement.
+ **/
+
+int main()
+{
+    int w; // width of the building.
```
Comment (bjones1) [inline ESD109-Shadowsedit/ShadowsOfTheKnight_all655.cpp:81]: Put comments on the preceeding line, so the CodeChat Editor will display them as a doc block.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by all655** [comment]
Due to an error that I experienced with making changes to the directory format, I have submitted a separate pull request (#37) that addressed your change suggestions. 

Sorry for the inconvenience.

**Pull request closed without merging**